src/@batch-flask/ui/form/form-field/form-field.scss

```diff
@@ -51,6 +51,7 @@ bl-form-field {
         width: 100%;
         flex: 1;
         min-width: 0;
+        border: 1px #000000 solid;
```

Comment:
Could we switch this hex to use theme colors instead (like `$border-color`)? If we need to use black for this border then we could keep it since `$border-color` is currently set to a dark gray.